File: Zero Judge Java/a006.java
import java.util.Scanner;
public class a006 {
    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);

        int a = scn.nextInt();
        int b = scn.nextInt();
        int c = scn.nextInt();
        
        int judge = b*b-4*a*c;
        int Positive = (-b+(int)Math.sqrt(b*b-4*a*c))/(2*a);
        int Negative = (-b-(int)Math.sqrt(b*b-4*a*c))/(2*a);

        if(judge>0){
            System.out.printf("Two different roots x1=%d , x2=%d",Positive,Negative);
        }
        else if(judge<0){
            System.out.println("No real root");
        }
        else{
            System.out.printf("Two same roots x=%d",Positive);
        }
        scn.close();
    }
}

File: Zero Judge Python/a006.py
import math
a,b,c = map(int,input().split())

judge = b*b-4*a*c

if(judge>0):
    Positive = int((-b+math.sqrt(b*b-4*a*c))//(2*a))
    Nagetive = int((-b-math.sqrt(b*b-4*a*c))//(2*a))
    print(f"Two different roots x1={Positive} , x2={Nagetive}")
elif(judge<0):
    print("No real root")
else:
    Positive = int((-b+math.sqrt(b*b-4*a*c))//(2*a))
    Nagetive = int((-b-math.sqrt(b*b-4*a*c))//(2*a))
    print(f"Two same roots x={Positive}")